libnmstate/nm/plugin.py

```diff
@@ -141,6 +141,9 @@ def get_interfaces(self):
             iface_info.update(get_infiniband_info(applied_config))
             iface_info.update(get_current_macvlan_type(applied_config))
 
+            if iface_info[Interface.TYPE] == InterfaceType.OVS_PORT:
+                continue
+
             if NmstatePlugin.OVS_CAPABILITY in capabilities:
```

Comment:
The fix is to replace this condition with `nm_ovs.has_ovs_capability(self.client)`.
We should not process the body only if the NM plugin is not installed, if the OVS is running or not is irrelevant.

Reply:
If OVS runnig is irrelevant why was it added ?

Reply:
OVS running check was added to the capabilities report and the condition was left as it.
Based on your findings and the input from the NM team, it may not be needed at all.